lib/elixir/test/elixir/file/stream_test.exs

```diff
@@ -141,6 +141,25 @@ defmodule File.StreamTest do
         end
       end
 
+      test "supports byte offset" do
```

Comment:
One test for offset+trim_bom and we are good to go!

Reply:
Added :)